scipy/stats/_bootstrap.py

```diff
@@ -0,0 +1,375 @@
+    method : str in {'percentile', 'basic', 'bca'}, optional
+        Whether to return the 'percentile' bootstrap confidence interval
+        (``'percentile'``), the 'reverse'or the bias-corrected and accelerated
+        bootstrap confidence interval (``'BCa'``). The default is ``'BCa'``.
+        Note that only ``'percentile'`` and ``'basic'`` support multi-sample
+        statistics at this time.
+    random_state : {None, int, `numpy.random.Generator`,
+                    `numpy.random.RandomState`}, optional
+
+        If `seed` is None (or `np.random`), the `numpy.random.RandomState`
+        singleton is used.
+        If `seed` is an int, a new ``RandomState`` instance is used,
+        seeded with `seed`.
+        If `seed` is already a ``Generator`` or ``RandomState`` instance then
+        that instance is used.
+
+        Pseudorandom number generator state used to generate resamples.
+
+    Returns
+    -------
+    ci_l : float
+        The lower bound of the confidence interval.
+    ci_u : float
+        The upper bound of the confidence interval.
+
+    References
+    ----------
+    .. [1] B. Efron and R. J. Tibshirani, An Introduction to the Bootstrap,
+       Chapman & Hall/CRC, Boca Raton, FL, USA (1993)
+    .. [2] Nathaniel E. Helwig, "Bootstrap Confidence Intervals",
+       http://users.stat.umn.edu/~helwig/notes/bootci-Notes.pdf
+    .. [3] Bootstrapping (statistics), Wikipedia,
+       https://en.wikipedia.org/wiki/Bootstrapping_(statistics)
+
+    Examples
+    --------
+    Suppose we have sampled data from an unknown distribution.
+
+    >>> import numpy as np
+    >>> rng = np.random.default_rng()
+    >>> from scipy.stats import norm
+    >>> dist = norm(loc=2, scale=4)  # our "unknown" distribution
+    >>> data = dist.rvs(size=100, random_state=rng)
+
+    We are interested int the standard deviation of the distribution.
+
+    >>> std_true = dist.std()      # the true value of the statistic
+    >>> print(std_true)
+    4.0
+    >>> std_sample = np.std(data)  # the sample statistic
+    >>> print(std_sample)
+    3.9460644295563863
+
+    We can calculate a 90% confidence interval of the statistic using
+    `bootstrap_ci`.
+
+    >>> from scipy.stats import bootstrap_ci
+    >>> data = (data,)  # samples must be in a sequence
+    >>> ci_l, ci_u = bootstrap_ci(data, np.std, confidence_level=0.9,
+    ...                           random_state=rng)
+    >>> print(ci_l, ci_u)
+    3.5636350108774204 4.371806172295983
+
+    If we sample from the distribution 1000 times and form a bootstrap
+    confidence interval for each sample, the confidence interval
+    contains the true value of the statistic approximately 900 times.
+
+    >>> n_trials = 1000
+    >>> ci_contains_true_std = 0
+    >>> for i in range(n_trials):
+    ...    data = (dist.rvs(size=100, random_state=rng),)
+    ...    ci = bootstrap_ci(data, np.std, confidence_level=0.9,
+    ...                       random_state=rng)
```

Comment:
I saw you changed this but seems like it was correct.
```suggestion
    ...                      random_state=rng)
```

Reply:
First there weren't `...` and then they weren't indented. (I was working in GitHub rather than editing locally - usually not a good idea.) When I fixed the indent, I must have remove three instead of four spaces from the right side of the `...`.